packages/components/src/components/modal/Modal.tsx

```diff
@@ -45,7 +45,7 @@ export const ModalFooter: React.FC<ModalContentProps> = ({
   ...rest
 }: ModalContentProps) => <div className={classNames(buildClass('footer'), className)} {...rest}> {children}</div >;
 
-const Modal: React.FC<ModalProps> = ({
+const Modal: React.FC<ModalProps> & React.HTMLProps<HTMLDivElement> = ({
```

Comment:
I think `size` should be omitted as it's a custom prop on the component.

Reply:
Good point! I will update it !